ct-app/core/components/messages.py

```
@@ -1,7 +1,5 @@
-import re
+import os
```

Ruff (0.8.0)

`os` imported but unused

Remove unused import: `os`

(F401)